### PR TITLE
[Java][V3][V4]fix font type issue

### PR DESCRIPTION
**Pull request by kaibocai**
Releasing new Host versions.
[V3 Release](PASTE_V3_LINK_HERE)
[V2 Release](PASTE_V2_LINK_HERE)

---

This PR fixes the [ICM issue](https://portal.microsofticm.com/imp/v3/incidents/details/300470002/home), when migrate to Eclipse openjdk8 and Microsoft openjdk11, they removed font type dependency files in their JRE to avoid dependency conflict. In our case we need to manually install font dependencies. 
More detail can be find at[Skip freetype build and bundling on Linux](https://github.com/adoptium/temurin-build/pull/2226), [prerequisites-for-font-support-in-adoptopenjdk](https://github.com/AdoptOpenJDK/blog/blob/ba5844ddc0b7e25d8ae49ac65a8b4e25dea5a48c/content/blog/prerequisites-for-font-support-in-adoptopenjdk/index.md#linux)

### PR information

- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [X] If applicable, the PR references the bug/issue that it fixes in the description.
- [X] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

**Review by karianna — APPROVED** [review]
That all looks correct to me.

File: host/3.0/buster/amd64/java/java11/java11-core-tools.Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM mcr.microsoft.com/java/maven:11-zulu-debian10
+FROM mcr.microsoft.com/azure-functions/java:3.0-java11-build
```
Comment (CooperLink) [inline host/3.0/buster/amd64/java/java11/java11-core-tools.Dockerfile:6]: Is this change scoped ?  It seems different than the others. I am not very concerned with it. just wondering if it should be here

**Review by CooperLink — APPROVED** [review]
This LGTM. Change is scoped to Java. The location of the additions shouldn't hurt layering.  Good job and thanks Kaibo

File: host/3.0/buster/amd64/java/java11/java11-core-tools.Dockerfile (hunk 1 above)
Comment (karianna) [reply host/3.0/buster/amd64/java/java11/java11-core-tools.Dockerfile:6]: Zulu is no longer a supported OpenJDK distribution for functions so I think that change is required as such, could perhaps be in a separate PR though?

Comment (kaibocai) [reply host/3.0/buster/amd64/java/java11/java11-core-tools.Dockerfile:6]: Thank you for bring this up @CooperLink @karianna . Yes, zulu is not a supported jdk for functions. For core tools v3 we didn't update it to use our own java-build image which migrate from zulu to adoptium last time. We are thinking wait for next release to update it. I can have a separate PR for it if this is preferred. Also I will attach local test results on the updated core tools images. Thanks a lot!

**Review by CooperLink — APPROVED** [review]

**Comment by kaibocai** [comment]
Tests v3 core tools on local 
Java11:
image
Java8:
image

**Pull request merged**